receiver/vcenterreceiver/metadata.yaml

```diff
@@ -73,6 +73,14 @@ attributes:
     enum:
       - virtual
       - physical
+  memory_usage_type:
+    name_override: type
+    description: type of memory usage within a resource pool.
```

Comment:
This could theoretically be applied to some other resource types down the road. It might make more sense to have the description as "The type of memory usage."